docs/bridged_networks.md

```diff
@@ -53,10 +52,6 @@ A wishlist of IRC networks to be bridged is being collected [in a github issue](
 
```

Comment:
Just for the sake of tradition.
```suggestion
* The Freenode IRC bridge offically was shut down on 20-Dec 2021.
```

Reply:
I changed the network name to lowercase as that is what they used to prefer and used ISO 8601 date format instead and for this suggestion made you a coauthor.